docs/Catalog/Preferences.md

```diff
@@ -53,6 +55,10 @@ ui:
 * `ui.actions.createPackage: False` - hide buttons to create packages via
 drag-and-drop or from folders in S3
 * `ui.actions.deleteRevision: True` - show buttons to delete package revision
+* `ui.actions.downloadObject: False` - hide buttons to download
+files and directories under "Bucket" tab
+* `ui.actions.downloadPackage: False` - hide buttons to download
+files and directories under "Packages" tab
```

Comment:
wouldn't this be sufficient?
```suggestion
* `ui.actions.downloadObject: False` - hide download buttons under "Bucket" tab
* `ui.actions.downloadPackage: False` - hide download buttons under "Packages" tab
```

Reply:
Probably, yes, but I wanted to be specific and express actual intention.